foundations/Cargo.toml

```diff
@@ -77,6 +77,12 @@ telemetry = [
     "telemetry-server",
 ]
 
+tokio-metrics = [
```

Comment:
Needs to be added in lib.rs docs describing features along with explanation of how to enable it (as it's different from the rest)